src/components/work/Footer/Footer.tsx

```diff
@@ -14,20 +14,35 @@ import {
 } from 'reactstrap';
 import { Link } from 'react-router-dom';
 
+import { SETTINGS } from 'configs/settings';
 import { BeenestSVGPrimary } from 'shared/svgComponents/SvgComponents';
 
+const { BEENEST_HOST } = SETTINGS;
+
 const contentData = [
   {
-    header: 'Account',
-    links: [['Profile', '/work'], ['User Contacts', '/work'], ['Projects', '/work'], ['Settings', '/work']],
+    header: 'Top Cities',
+    links: [
+      ['San Francisco', `${BEENEST_HOST}/listings?locationQuery=San%20Francisco%2C%20CA%2C%20USA`],
```

Comment:
can you use the `/markets/san-francisco` alias?

Reply:
nice didn't know we had that